kivy/tests/test_uix_translate_coordinates.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+import functools
+
+NON_RELATIVE_TYPE_WIDGET_CLS_NAMES = ('Widget', )
+RELATIVE_TYPE_WIDGET_CLS_NAMES = (
```

Comment:
Can you please make them lowercase? That fits in better with our and Python's style.

Reply:
ok done